packages/amplication-client/src/Workspaces/MemberListItem.scss

```
@@ -18,19 +18,18 @@
   }
   &__description {
     @include body2;
-    color: var(--black80);
+    color: #53dbee;
   }
 
   &__property {
-    color: var(--positive-default);
-    border: 1px solid var(--positive-default);
-    height: 20px;
-    border-radius: 10px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    padding: 0 var(--default-spacing);
-    background-color: rgba(49, 197, 135, 0.09);
+    font-family: Poppins;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #ffbd70;
```

```suggestion
  @include regular-12
```

I'm unsure about the color, but if what we have in regular-12, is not what you need, you can override it with the color property, just use vars, for example `color: var(--black80);`